### ISSUE TITLE
Point-and-click interface to run validation tests

### ISSUE THREAD

**Issue by apdavison** [cnr-ibf-pa/hbp-bsp-issues#84]

**Comment by alex4200** [comment]
@apdavison  : can you please assign this issue to a month-milestone? Thanks

**Comment by apdavison** [comment]
This is for SGA2

**Comment by alex4200** [comment]
@apdavison Can you assign an assignee?

**Comment by appukuttan-shailesh** [comment]
@apdavison : I believe the milestone can be moved to M22. Unlikely to be undertaken before Sep 2019.
Also, is this intended to employ the Jupyter widgets that are to be developed in the voucher call. Is the specified deadline of M21 (Dec 2019) a hard deadline?

**Comment by apdavison** [comment]
@appukuttan-shailesh no, I think this will be built into the Model Validation Collaboratory app. M21 is not a hard deadline, this feature is not written explicitly into the SGA2 workplan.

I've updated the description and milestone accordingly.

**Comment by apdavison** [comment]
This will not be done in SGA2.
